main.yml

```diff
@@ -299,6 +299,8 @@ policies:
   channel: '#cloudservices-policies'
   deployment_repo: https://github.com/RedHatInsights/policies-ui-frontend-build
   frontend:
+    module: policies#./RootApp
+    section: security
```

Comment:
Sections are not supported on prod-beta. Should be just the module definition.

Reply:
upps forgot to add the commit